packages/ui/components/dialog/Dialog.tsx

```diff
@@ -109,6 +113,9 @@ export const DialogContent = React.forwardRef<HTMLDivElement, DialogContentProps
     return (
       <Portal>
         <Overlay className="fadeIn fixed inset-0 z-50 bg-neutral-800 bg-opacity-70 transition-opacity dark:bg-opacity-70 " />
+        {modalDisabled && (
```

Comment:
How about we call it `forceOverlayWhenNoModal` to make it clear its use.

i would have preferred to be able to use Dialog.modal prop here using Dialog context but that seems to be not provided by radix-ui.

Reply:
Okay 👍, nah it's not provided by them.